DotNetCoreMasters/DomainModel/ItemDomainModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace DomainModel
+{
+    public class ItemDomainModel
```

Comment:
This should be named "Item". You can leave as is since you used it correctly but we suggest you to rename it for ease of use.

Reply:
I'll leave as is, for me it is easier to understand that it is from Domain Mopdel